src/layouts/header.html

```diff
@@ -1,4 +1,4 @@
-<header class="w-full border-b border-gray-200 font-openSans">
+<header id="header" class="w-full border-b border-gray-200 font-openSans sticky top-0 bg-white">
```

Comment:
```suggestion
<header class="w-full border-b border-gray-200 font-openSans sticky top-0 bg-white">
```
id header này bên ngoài thành phần khi gọi đã có rồi, nên mình sẽ k viết lại để khỏi bị trùng Tỉnh nha